src/main/java/com/dell/isg/smi/service/server/inventory/controller/ServerInventoryController.java

```diff
@@ -109,9 +108,7 @@ public HwSystem summary(@RequestBody Credential payload) {
 
 
     @RequestMapping(value = "/software", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
-    @ApiOperation(value = "/software", nickname = "software", notes = "This operation allow user to get complete server software inventory throu wsman.", response = DCIMSoftwareIdentityType.class, responseContainer = "List")
-    // @ApiImplicitParams({
-    // @ApiImplicitParam(name = "credential", value = "Credential", required = true, dataType = "Credential.class", paramType = "Body", defaultValue = "no default") })
+    @ApiOperation(value = "/software", nickname = "software", notes = "This operation allow user to get complete server software inventory using wsman protocol.", response = DCIMSoftwareIdentityType.class, responseContainer = "List")
```

Comment:
"This operation allows the user to get complete server software inventory via the iDRAC using the WSMan protocol.